rbac/server/api/roles.py

```diff
@@ -14,6 +14,7 @@
 # ------------------------------------------------------------------------------
 """Roles APIs."""
 from uuid import uuid4
+import json as json1
```

Comment:
You imported json and called it json1 here, but not in the other files, is there a reason why you did this?

Reply:
Yes, because there is another json related package - 
"**from sanic.response import json**" which will not have the functions needed for my changes.

Reply:
Can I resolve this?

Reply:
When possible we should import the specific functions we need rather than the whole package. Maybe try `from json import dumps`?